### DISCUSSION TITLE
reference/docs/git

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#600] [Q&A]
# reference/docs/git

http://bbs.laoleng.vip/reference/docs/git.html

**Comment by Hooligan1209** [comment]
删除本地存在远程不存在的分支

$ git remote prune origin

**Reply by jaywcjlove** [reply]
已更新 @Hooligan1209

**Comment by geniyangge** [comment]
生成补丁
$ git format-patch -1 \<commit-SHA\>
-1 选项指定只生成一个补丁。\<commit-SHA> 是您要生成补丁的 Git 提交哈希。

应用补丁
git apply \<patch-file\>

**Comment by lx1186541801** [comment]
1

**Comment by leelenkj** [comment]
GIT配置不用每次都进行账号密码输入的方式：
Linux系统如下：
touch ~/.netrc  # 创建文件（如果不存在）
chmod 600 ~/.netrc  # 设置正确的权限
vim ~/.netrc  # 使用vim编辑（也可以用其他编辑器）
.netrc 内容配置如下：
machine github.com
  login your_github_username
  password your_github_token

machine gitlab.com
  login your_gitlab_username
  password your_gitlab_token

windows配置方式：
创建 .netrc 文件
在文件资源管理器中导航到用户主目录（通常是 C:\Users\你的用户名）
创建一个名为 _netrc 的文件（注意前面的下划线，这是 Windows 上的标准命名）
如果你使用的是 Git Bash，可以直接创建 .netrc 文件，路径为 C:\Users\你的用户名\.netrc
